src/field/crypto_bigint_monty.rs

```diff
@@ -24,9 +24,70 @@ pub struct MontyField<const LIMBS: usize>(MontyForm<LIMBS>);
 
 impl<const LIMBS: usize> MontyField<LIMBS> {
     /// Creates a new `MontyField` from a `MontyForm`.
+    #[inline(always)]
     pub const fn new(form: MontyForm<LIMBS>) -> Self {
         Self(form)
     }
+
+    #[inline(always)]
+    pub const fn new_unchecked(inner: Uint<LIMBS>, config: &MontyParams<LIMBS>) -> Self {
+        Self(MontyForm::from_montgomery(inner.into_inner(), *config))
+    }
+
+    #[inline(always)]
+    pub const fn inner(&self) -> &Uint<LIMBS> {
+        Uint::new_ref(self.0.as_montgomery())
+    }
+
+    #[inline(always)]
+    pub const fn into_inner(self) -> Uint<LIMBS> {
+        Uint::new(self.0.to_montgomery())
+    }
+
+    /// Retrieves the integer currently encoded in this [`ConstMontyForm`],
```

Comment:
(Here and below)
Comments are incorrect, this is `MontyForm`, not `ConstMontyForm`